botocore/model.py

```diff
@@ -707,6 +707,26 @@ def _get_streaming_body(self, shape):
                 return payload_shape
         return None
 
+    @CachedProperty
+    def has_modeled_body_input(self):
```

Comment:
Is this being used anywhere?  If not, we should hold off on adding it until it's needed